### DISCUSSION TITLE
The 'Add-PnPStoredCredential' command was found in the module 'PnP.PowerShell', but the  module could not be loaded

### DISCUSSION THREAD

**Discussion by NiuBClass666** [pnp/powershell#2964] [Q&A]
Add-PnPStoredCredential : The 'Add-PnPStoredCredential' command was found in the module 'PnP.PowerShell', but the 
module could not be loaded. For more information, run 'Import-Module PnP.PowerShell'.

We did not have this error by using 1.12.0 pnp module but now we get it with 2.1.1. Does this mean that we need to run 'Import-Module PnP.PowerShell' every time in the future after install-module PnP.PowerShell?

OS: Windows Server 2022 latest
Code:
          Install-Module -Name "PnP.Powershell" -Force

          $username = "$(BuildServicesAccount)"
          $password = (ConvertTo-SecureString -String "$(BuildServicesPassword)" -AsPlainText -Force)
          Add-PnPStoredCredential -Name https://myprojecy.sharepoint.com/ -Username $username -Password $password
          Connect-PnPOnline -Url "https://myprojecy.sharepoint.com/sites"

Update:
Adding 'Import-Module PnP.PowerShell' after 'Install-Module -Name "PnP.Powershell" -Force' gave us the following error:

Import-Module : Could not load file or assembly 'System.Management.Automation, Version=7.2.0.0, Culture=neutral, 
PublicKeyToken=31bf3856ad364e35' or one of its dependencies. The system cannot find the file specified

Solution:
It turns out that PnP Module 2.1.1 does not suppport powershell 5.x anymore. Using Powershell 7.x fixed our problem.

**Comment by arthurnardy** [comment]
Same error, magically appears over the week-end after having worked for months without change in the script -- currently looking for an explanation

**Comment by NiuBClass666** [comment]
Solution in edited Question section.

**Reply by arthurnardy** [reply]
I was precisely suspecting this explanation and have been waiting for a update toward 7.x powershell from it colleagues with administratior's rights. Thanks for the confirmation in advance!
